templates/faqs.html

```diff
@@ -59,7 +59,7 @@
 											<h4><span>Q.</span>I want to volunteer for the conference. What should I do?</h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>Please share your details by filling this form and the team will soon get back to you with responsibilities according to the domains you choose.</p>
+													<p><span>A.</span>Please share your details by filling this <a href = " http://bit.ly/inpycon2017">form</a> and the team will soon get back to you with responsibilities according to the domains you choose.</p>

```

Comment:
Please resolve conflicts with the base master branch.

If possible, do a rebase.